### PR TITLE
Style guide points

### PR DESCRIPTION
**Pull request by tyoc213**
Some extra specific points on style guide

### CODE REVIEW

File: docs_src/style-guide.md [hunk 1]
```diff
@@ -95,4 +95,12 @@ def dihedral(x:TensorImage,
...
\ No newline at end of file
+```
+
+## Notes
+
+* Don't use `typing` library.
+* Arguments should have one indent relative to method definition.
+* Only `cls` and `self` should be on same line of method name and first argument of `@patch` like `x` on `dihedral` sample.
+* Closing parenthesis should be aligned with `def`.
+* Keyword arguments (`**kwags`) should not be documented. (NOTE: not needed for the sprint, but see if `@delegates` can be used to automatically document keyword args).
```
Comment (muellerzr) [inline docs_src/style-guide.md:106]: Remove, I already can :)

**Review by muellerzr — COMMENTED** [review]
One quick nit!

File: docs_src/style-guide.md (hunk 1 above)
Comment (muellerzr) [reply docs_src/style-guide.md:106]: Spec the (NOTE)

**Review by warner-benjamin — APPROVED** [review]
Looks good pending resolution of Zach's comments.

File: docs_src/style-guide.md (hunk 1 above)
Comment (tyoc213) [reply docs_src/style-guide.md:106]: Q: "spec the NOTE" means remove the note?

Also what you mean with "remove, I already can"? I'm not so sure

Comment (tyoc213) [reply docs_src/style-guide.md]: I ermoved the note! :)

**Comment by tyoc213** [comment]
@warner-benjamin fixed :)

**Comment by muellerzr** [comment]
Great job!

**Pull request merged**